### ISSUE TITLE
[ISSUE] The `content` field in `workspace.Import` structure is marked as optional, leading to inability to import empty files

### ISSUE THREAD

**Issue by alexott** [databricks/databricks-sdk-go#696]
**Description**

The [content](https://github.com/databricks/databricks-sdk-go/blob/main/service/workspace/model.go#L390) field is marked as optional, so Terraform provider isn't able to create empty files, such as, `__init__.py`.

**Expected behavior**

I should be able to create empty files in the workspace

**Comment by mgyucht** [comment]
Thanks for raising this. Seems like a bug in the OpenAPI spec for this service. Will follow-up with the respective team.

**Comment by mgyucht** [comment]
@alexott is it possible to set `ForceSendFields: "Content"`? That should ensure that that field is serialized, even if `content` is an empty string.

**Comment by alexott** [comment]
Yes, I did it in TF that way, but we'll need to handle it in SDKs as well - I'm not sure why content is marked as optional
